docker/compose/demo/compose.yml

```diff
@@ -0,0 +1,128 @@
+version: "2.4"
+
+services:
+
+  dev_dataverse:
+    container_name: "dev_dataverse"
+    hostname: dataverse
+    image: gdcc/dataverse:alpha
+    restart: on-failure
+    user: payara
+    environment:
+      DATAVERSE_DB_HOST: postgres
+      DATAVERSE_DB_PASSWORD: secret
+      DATAVERSE_DB_USER: dataverse
+      DATAVERSE_FEATURE_API_BEARER_AUTH: "1"
```

Comment:
I left the bearer auth stuff in just in case someone wants to demo the new frontend as well. Should I note this under the security section?